src/server.ts

```diff
@@ -80,6 +82,13 @@ app.get('/otomi/values', async (req: Request, res: Response) => {
   }
 })
 
+app.get('/apl/schema', async (req: Request, res: Response) => {
+  const schema = await loadYaml(`${rootDir}/values-schema.yaml`)
+  const derefSchema = await $RefParser.dereference(schema as $RefParser.JSONSchema)
```

Comment:
would it make sense to load it once during the app init and serve z `derefSchema` from memory?

Reply:
I think we should first get the POC merged and later look at efficiency for loading the values-schema together with loading, calling and presenting in APL api

Reply:
please add this as a requirement to the ticket.